velox/dwio/parquet/tests/ParquetTpchTest.cpp

```diff
@@ -23,13 +23,23 @@
 #include "velox/functions/prestosql/registration/RegistrationFunctions.h"
 #include "velox/parse/TypeResolver.h"
 
+#include <map>
```

Comment:
Are these needed?

Reply:
my mistake, it should be `unordered_map`, some other header must be including this, hence it is not being needed. Should I avoid including it explicitly ?

Reply:
> Should I avoid including it explicitly ?

This would be nice.

Reply:
Okay, removed it but I don't think it is good thing in general to depend on some upstream header to include it.

Reply:
Thank you. 

> I don't think it is good thing in general to depend on some upstream header

I agree overall. However, it is impractical to include ALL headers used in a given file as that would require including a lot of headers from standard library for example.